spec/std/isa/inst/Zvks/vsm4r.vs.yaml

```
@@ -10,7 +10,10 @@ long_name: No synopsis available
 description: |
   No description available.
 definedBy:
-  anyOf: [Zvks, Zvksed]
+  extension:
+    anyOf:
+      - name: Zvks
+      - name: Zvksed
```

Zvks requires Zvksed.

```suggestion
  extension:
    name: Zvksed
```